packages/react/src/themes/font-awesome/components/Icon/iconStyles.ts

```diff
@@ -1,19 +0,0 @@
-import { ComponentSlotStylesInput, ICSSInJSStyle, ThemeIconSpec } from '../../../types'
-import { IconProps } from '../../../../components/Icon/Icon'
-import { IconVariables } from '../../../teams/components/Icon/iconVariables'
-import { emptyIcon } from '../../../base/components/Icon/iconNames'
-
-const iconStyles: ComponentSlotStylesInput<IconProps, IconVariables> = {
```

Comment:
Let's not delete font-awesome theme just yet.  Also, let's not port FA theme info to Teams theme.  Keep them separate.